modules/ROOT/pages/metadata.adoc

```diff
@@ -17,6 +17,8 @@ Integration developers often spend a lot of time trying to determine the input a
 
 DataSense uses the metadata provided by these components to resolve all this information automatically and present it to your end user in design time, which drastically increases the speed of development.
 
+Note that the main purpose of this feature is not to give the end user of the connector a list of possible values for a configuration. To solve that use case we recommend to implement a xref:value-providers.adoc[Value Providers].
```

Comment:
I would add something like "Unlike Value Providers which are meant to narrow down the possible values that a parameter of a KNOWN type can take, this feature is to discover dynamic types which universe of possible values is not to be discovered by this feature".

Un equivalente (but reverse) statement should be added on the value providers side

Reply:
Agree with Mariano